corehq/messaging/scheduling/migrations/0026_add_model_fcm_notification_content.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 3.2.19 on 2023-06-01 17:44
+
+from django.db import migrations, models
+import django.db.models.deletion
+import jsonfield.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('scheduling', '0025_schedule_and_broadcast_deleted_on'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FCMNotificationContent',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
just confirming if other Content models also use auto generated ID db field? In the past we have known to exhaust the limit for an integer ID field and had to update it to Long or use UUID instead if the model isn't linked to another model.

Reply:
Yes that is correct - other Content models also use auto generated ID db field.